spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+
+ describe "POST #create" do
+
+    context "validation" do
```

Comment:
There are a lot of things going on here:
1) You should start or create the tests in the same order as in the controller, just to have the code organized
2) The context string should start with "when", for example, `context "when the request is invalid"`
3) There are a lot of spaces between each line, it is unnecessary
4) You are not testing all the actions from the controller, you are missing the `update`
5) It is "invalid" not "unvalid" (line 9)
6) It is recommended to use `be_falsy` or `be_truthy` when testing booleans (lines 16 and 27)